exporter/elasticsearchexporter/config.go

```diff
@@ -160,12 +160,12 @@ type MappingsSettings struct {
 	// Mode configures the field mappings.
 	Mode string `mapstructure:"mode"`
 
-	// Try to find and remove duplicate fields
+	// Dedup is non-operational, and will be removed from January 2025.
```

Comment:
Maybe let's not commit to dates?
```suggestion
	// Dedup is non-operational, and will be removed in the future.
```

Reply:
The intention here is to indicate the earliest time at which the config will be removed. I'm not too bothered though.